src/devices/cpu/t11/t11.h

```diff
@@ -19,7 +21,7 @@ enum
 #define T11_IRQ3        3      /* IRQ3 */
 
 
-class t11_device :  public cpu_device
+class t11_device :  public cpu_device, public z80_daisy_chain_interface
```

Comment:
Does the T11 actually implement the Z80 interrupt interface?

Reply:
The T11 interrupt interface has practically nothing in common with the Z80 interrupt interface besides being able to either autovector interrupts or have an 8-bit vector delivered through the bus. (This is actually unlike older PDP-11s, which lack autovectoring except for special nonmaskable conditions and software-generated PIRQs on models that support those, and receive vectors as 16-bit words.)

Reply:
Yeah, it's a hack, although QBUS interrupt interface does have something in common (BIAKI/BIAKO signals serve much the same purpose as IEI/IEO).